src/app/VerifyActivities/components/VerifyActivitiesContainer.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import dashboardActions from '../../Dashboard/operations/actions';
+import { actions } from '../../Societies/operations';
+import { ButtonComponent } from '../../common/components';
+import VerifyActivities from './VerifyActivitiesComponent';
+import { SocietyStatsComponent } from '../../Dashboard/components';
+import { getUserInfo, getToken } from '../../utils/tokenIsValid';
+import ACTIVITY_STATUS from '../constants';
+
+export class VerifyActivitiesContainer extends Component {
+  static defaultProps = {
+    society: {},
+    societyName: '',
+    fetchUserActivites: null,
+    fetchSocietyInfoRequest: null,
+  };
+
+  static propTypes = {
+    society: PropTypes.shape({}),
+    societyName: PropTypes.string,
+    fetchUserActivites: PropTypes.func,
+    fetchSocietyInfoRequest: PropTypes.func,
+  };
+
+  componentDidMount() {
+    const { fetchSocietyInfoRequest, societyName, fetchUserActivites } = this.props;
+    const token = getToken();
+    const userInfo = getUserInfo(token);
+    fetchUserActivites(userInfo.id);
+    fetchSocietyInfoRequest(societyName.toLowerCase());
+  }
+
+  componentDidUpdate(prevProps) {
+    const { fetchSocietyInfoRequest, societyName } = this.props;
+    if (prevProps.societyName !== societyName) {
+      fetchSocietyInfoRequest(societyName.toLowerCase());
+    }
+  }
+
+  filterActivitiesByInReviewStatus = activities => (
+    activities.filter(activity => activity.status === ACTIVITY_STATUS.IN_REVIEW))
+
+  render() {
+    const { society, societyName } = this.props;
+    let verifyActivitiesHtml = <p>Loading ...</p>;
```

Comment:
should we have a loader here instead of plain text

Reply:
Very true. We can create a story for this since I have the same on the dashboard.

Reply:
I created a `LoadingComponent` @Kachulio1